test/distributed/_tensor/test_embedding_ops.py

```diff
@@ -123,10 +138,10 @@ def test_sharded_embedding_colwise(self):
         self._run_embedding_op_test(1, [8, 6, 5, 4], 23, 13, padding_idx=12)
 
     @with_comms
-    def test_sharded_embedding_colwise_errors(self):
+    def test_sharded_embedding_colwise_max_norm_errors(self):
```

Comment:
Are colwise and rowwise embedding sharding among the major feature gaps? Are they easy to implement? What's the main blocker?

Reply:
embedding norm is kind of tricky I think, it probably requires reduction, I haven't looked into what's the gap there, feel free to explore this!